app/views/templates/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= javascript_tag("$(document).on('ContentLoad', tfm.templateInputs.initTypeChanges)"); %>
+<%= javascript_tag("$(document).on('ContentLoad', tfm.templateInputs.initTypeChanges, tfm.templateKind.initTemplateKind)"); %>
```

Comment:
This breaks `ptable` edit as for ptable there is no kind available.
I think this initialization has to be done in `views/provisioning_templates/_custom_tabs.html.erb`